postgresmodels/posture.go

```diff
@@ -0,0 +1,95 @@
+package postgresmodels
+
+import (
+	"time"
+
+	"github.com/lib/pq"
+	"gorm.io/datatypes"
+)
+
+type PostureReportStatus struct {
+	BaseModel
+	ReportGUID string `gorm:"primaryKey"`
+	BaseReport
+}
+type ClusterPostureReport struct {
+	BaseModel
+	ReportGUID               string `gorm:"primaryKey"`
+	ClusterGUID              string
+	ClusterName              string
+	CustomerGUID             string
+	Score                    float32
+	Timestamp                time.Time
+	WorkerNodeCount          int
+	KubescapeVersion         string
+	KubernetesVersion        string
+	HelmChartVersion         string
+	RegoLibraryVersion       string
+	TotalControls            int
+	FailedControls           int
+	SkippedControls          int
+	CriticalSeverityControls int
+	HighSeverityControls     int
+	MediumSeverityControls   int
+	LowSeverityControls      int
+}
+
+type ControlScanResult struct {
+	BaseModel
+	ControlID                  string `gorm:"primaryKey"`
+	ReportGUID                 string `gorm:"primaryKey"`
+	FrameworkName              string `gorm:"primaryKey"`
+	Name                       string
+	Status                     string
+	SubStatus                  string
+	StatusCode                 int
+	ComplianceScore            float32
+	AffectedResourcesCount     int
+	FailedResourcesCount       int
+	SkippedResourcesCount      int
+	WarningResourcesCount      int
+	TotalScannedResourcesCount int
+}
+
+type FrameworkSummary struct {
```

Comment:
comment why we need it, since we can calculate it using the ControlScanResult records